src/include/bicop/class.hpp

```diff
@@ -1,141 +1,183 @@
 #pragma once
 
 #include "docstr.hpp"
-#include <pybind11/eigen.h>
-#include <pybind11/pybind11.h>
-#include <pybind11/stl.h>
+#include <nanobind/eigen/dense.h>
+#include <nanobind/nanobind.h>
 #include <vinecopulib.hpp>
 
-namespace py = pybind11;
+namespace nb = nanobind;
+using namespace nb::literals;
 using namespace vinecopulib;
 
+// Factory function to create a Bicop from family, rotation, parameters, and
+// variable types
+inline Bicop
+from_family(const BicopFamily& family,
```

Comment:
shouldn't it be `bc_from_family()` in view of the function names below?

Reply:
* Will update.

Reply:
all good otherwise, can merge